.github/workflows/e2e-test.yaml

```
@@ -0,0 +1,36 @@
+name: End to End Test
+on: push
+
+jobs:
+  db-job:
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres
```

I would pin the image to a specific version.
```suggestion
        image: postgres:17
```